recipes/intel-repack/meta.yaml

```diff
@@ -1,6 +1,8 @@
 {% set version = "2019.5" %}
 {% set buildnum = "281" %}
 
+{% set my_channel_targets = channel_targets if channel_targets is defined else ['conda-forge', 'defaults'] %}
```

Comment:
I didn't realize that `is defined` was valid jinja2 syntax, awesome!

Reply:
I need to file a smithy bug report.

Reply:
Can you please link the issue here once you file it?

Reply:
Yeah for sure! The basic problem is that channel_targets is not defined for some of the recipe handling somewhere.